packages/embed-react/src/utils/nonce.ts

```diff
@@ -1,9 +1,6 @@
 declare global {
-  namespace NodeJS {
-    interface Global {
-      __webpack_nonce__: string
-    }
-  }
+  // eslint-disable-next-line no-var
+  var __webpack_nonce__: string
```

Comment:
Can this be defined as `let` instead?
```suggestion
  let __webpack_nonce__: string
```

Reply:
I tried with let but then TS complains that it cannot find `__webpack_nonce__`. I guess the globals have to use `var`.